<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fdog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="libres.qrc">
    <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:Microsoft YaHei;}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>601</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget{

background-color: rgb(193, 255, 183);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>4</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="icon_tool">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>33</width>
             <height>33</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="skin_tool">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/skin.png</normaloff>:/lib/skin.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="min_tool">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/zuixiaohua.png</normaloff>:/lib/zuixiaohua.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="close_tool">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/guanbi.png</normaloff>:/lib/guanbi.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>301</width>
           <height>87</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>301</width>
           <height>93</height>
          </size>
         </property>
         <widget class="QLabel" name="signature_label">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">:hover{
border-style:solid;
border-width:1px;
	border-color: rgb(108, 108, 108);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_2">
          <property name="geometry">
           <rect>
            <x>49</x>
            <y>47</y>
            <width>18</width>
            <height>18</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="libres.qrc">
            <normaloff>:/lib/zaixian.png</normaloff>:/lib/zaixian.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="username_label">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>16</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>10</y>
            <width>53</width>
            <height>53</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">#pushButton{
border-radius:25px;
border-style:solid;
border-width:2px;
border-color: rgba(255, 255, 255,20);
}
#pushButton::hover{
border-radius:25px;
border-style:solid;
border-width:4px;
border-color: rgb(255, 255, 255);
}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_5">
          <property name="geometry">
           <rect>
            <x>142</x>
            <y>13</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 47, 11);
border-image: url(:/lib/dengji.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_6">
          <property name="geometry">
           <rect>
            <x>195</x>
            <y>13</y>
            <width>40</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="libres.qrc">
            <normaloff>:/lib/svip.png</normaloff>:/lib/svip.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
         <zorder>signature_label</zorder>
         <zorder>username_label</zorder>
         <zorder>pushButton</zorder>
         <zorder>toolButton_5</zorder>
         <zorder>toolButton_6</zorder>
         <zorder>toolButton_2</zorder>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,22);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/search.png</normaloff>:/lib/search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>272</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgba(0,0,0,22);border-color:rgba(0,0,0,255);color: rgba(9, 0, 3,0);border-style:none;border-width:1px;border-radius:0px;font:15px &quot;黑体&quot;;</string>
           </property>
           <property name="placeholderText">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>318</width>
           <height>384</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget::tab{
background-color: rgb(255, 255, 255);
border-style:none;
border-width:1px;
border-radius:0px;
font:18px &quot;黑体&quot;;
height:38px;
width:105px;
	color: rgb(100, 100, 100);
}
</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>消息</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">width:10px;</string>
             </property>
             <property name="autoScrollMargin">
              <number>16</number>
             </property>
             <property name="batchSize">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>联系人</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="styleSheet">
              <string notr="true">QScrollBar:vertical{
background-color:rgba(0,0,0,0);
border-color:rgba(0,0,6,0);
border-style:solid;
border-width:0px;
width:10px;
padding-top:0px;
padding-bottom:0px;
}


QScrollBar::handle:vertical{
border-style:solid;
border-width:2px;
border-radius:4px;
background-color:rgba(145,145,145,255);
border-color:rgba(0,0,0,0);
min-height:44px;}


QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,0);}


QScrollBar::sub-page:vertical,QScrollBar::sub-page:vertical{background:rgba(0,0,0,0);}</string>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>310</width>
                <height>396</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">#scrollAreaWidgetContents{
background:transparent;
}</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>空间</string>
          </attribute>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>311</width>
             <height>411</height>
            </rect>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>309</width>
              <height>409</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/sangangy.png</normaloff>:/lib/sangangy.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="libres.qrc">
             <normaloff>:/lib/tianjiahaoyou.png</normaloff>:/lib/tianjiahaoyou.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="15" margin="15"/>
 <resources>
  <include location="libres.qrc"/>
 </resources>
 <connections/>
</ui>
